x-pack/legacy/plugins/code/server/queue/abstract_git_worker.ts

```diff
@@ -27,12 +31,34 @@ export abstract class AbstractGitWorker extends AbstractWorker {
     protected readonly log: Logger,
     protected readonly client: EsClient,
     protected readonly serverOptions: ServerOptions,
-    protected readonly gitOps: GitOperations
+    protected readonly gitOps: GitOperations,
+    protected readonly watermarkService: DiskWatermarkService
   ) {
     super(queue, log);
     this.objectClient = new RepositoryObjectClient(client);
   }
 
+  public async executeJob(_: Job): Promise<WorkerResult> {
+    const { thresholdEnabled, watermarkLowMb } = this.serverOptions.disk;
+    if (thresholdEnabled) {
+      const isLowWatermark = await this.watermarkService.isLowWatermark();
+      if (isLowWatermark) {
+        const msg = i18n.translate('xpack.code.git.diskWatermarkLow', {
```

Comment:
it need to have a suffix, here I think it's `xpack.code.git.diskWatermarkLowMessage`

Reply:
done